tools/stimulus/types-RR.yaml

```diff
@@ -297,7 +297,7 @@ VERTEXSET_LIST:
 
 VERTEXSETLIST_INT:
     INCONV:
-        IN: '%I% <- lapply(%I%, function(x) as.integer(x)-1L)'
+        IN: '%I% <- lapply(%I%, function(x) as.numeric(x)-1)'
```

Comment:
This looks like a bug fix?

Reply:
This is a readability fix. Now we use doubles for indices, so we should be consistent, and avoid integers. Whenever I see `1L` I'm thinking about whether there's a corresponding `int` on the C side that can overflow or otherwise prevent large indices.

Reply:
This was a mistake on my part. `VERTEXSETLIST_INT` is unused and should be completely removed.

Reply:
See #1049